articles/enroll-personal-byod-ios-ipad-hosts-with-managed-apple-account.md

```diff
@@ -0,0 +1,82 @@
+
+## Step 3: Connect (federate) your identity provider (IdP) with Apple Business Manager (ABM)
+
+Follow the [Apple documentation](https://support.apple.com/en-gb/guide/apple-business-manager/axmb19317543/web) to connect your identity provider (IdP). This will enable end users to log in to their Managed Apple Account using their existing IdP credentials.
+
+You can watch these videos as well:
+ - [Connect Google Workspace to ABM](https://www.youtube.com/watch?v=CPfO6W67d3A)
+ - [Connect Microsoft Entra ID to ABM](https://www.youtube.com/watch?v=_-PnhMurAVk)
+
+## Step 4: Create a `📱🔐 Personal mobile devices` team.
+
+Fleet's [best practice](https://fleetdm.com/guides/teams#best-practice) is to create team for personal hosts that have access to a company resources.
+
+In this team you can add custom OS settings that are compatible with hosts enrolled with Acount-driven User Enrollment. To find out which payloads are compatible with User Enrollment, visit the [Apple documentation](https://support.apple.com/en-gb/guide/deployment/dep6ae3f1d5a/1/web/1.0).
+
+## Step 5: Login to enroll to Fleet (end user experience)
+
+Ask your end users to go to **Settings > General > VPN & Device Management > Sign In to Work or School Account...** and log in using their IdP credentials.
+
+When they login, their iOS/iPadOS host will enroll to Fleet.
+
+## Self-host a service discovery file (well-known resource)
+
+
+If your iOS/iPadOS hosts are running iOS 18.2/iPadOS 18.2 and later, you can skip need this. Fleet manages service discovery for hosts running iOS 18.2/iPadOS 18.2 and later. 
+
+If your iOS/iPadOS hosts are below iOS 18.2/iPadOS 18.2 or you're trying Fleet, you have to self-host a [service discovery JSON file](https://support.apple.com/en-gb/guide/deployment/dep4d9e9cd26/web#depcae01b5df).
+
+If you're trying Fleet and using a different MDM solution in production, hosting this file will direct only Account-driven user enrollments to the Fleet. iOS/iPadOS hosts purchased in ABM and hosts that use an enrollment profile will still enroll to your current MDM solution.
+
+Host this JSON file below at the following URL: `https://<company_domain>/.well-known/com.apple.remotemanagement.`
+
+Make sure the `Content-Type` header is set to `application/json`.
+
+```json
+{
+  "Servers": [
+    {
+      "Version": "mdm-byod",
+      "BaseURL": "https://<fleet_server_url>/api/mdm/apple/account_driven_enroll"
+    }
+  ]
+}
+```
+
+## Inventory limitations
+
+- Fleet does not have access to the serial numbers of personal hosts due to Apple's privacy limitations.
+- For personal hosts, Fleet can only inventory applications from the work profile.
```

Comment:
```suggestion
- For personal hosts, Fleet can only inventory applications installed by Fleet.
```

I like @allenhouchins's suggestion. The language cross-platform. I switched "MDM" to "Fleet"